### PR TITLE
Added ArrayFire-Nim to packages.json

### PR DESCRIPTION
**Pull request by bitstormGER**
Hi,

I have publishes version 0.1.0 of ArrayFire-Nim - a wrapper for the ArrayFire Library.
This is the first time I try to publish on the official package list - hopefully I got everything right,

Best Regards

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -5618,5 +5618,18 @@
     "description": "Simple extensible implementation of Porter stemmer algorithm",
     "license": "MIT",
     "web": "https://github.com/iourinski/porter"
+  },
+  {
+    "name": "arrayfire-nim",
```
Comment (dom96) [inline packages.json]: Remove the dash from the name please. This needs to match the name of your .nimble file which is ``ArrayFireNim`` (the case doesn't matter though).

**Comment by bitstormGER** [comment]
Should be ok now

File: packages.json [hunk 2]
```diff
@@ -5618,5 +5618,31 @@
     "description": "Simple extensible implementation of Porter stemmer algorithm",
     "license": "MIT",
     "web": "https://github.com/iourinski/porter"
+  },
+  {
```
Comment (dom96) [inline packages.json]: How come the diff is showing you added this?

**Comment by bitstormGER** [comment]
I merged and removed the diff marker because I had a conflict, sorry if I messsed up, do you want me to redo the pull request?

**Comment by dom96** [comment]
Merged manually via https://github.com/nim-lang/packages/commit/92605a3032417d6019577d4a873d38f02423216b

**Pull request closed without merging**